eosclubhouse/clubhouse.py

```diff
@@ -3082,6 +3083,7 @@ def __init__(self):
         super().__init__(application_id=CLUBHOUSE_NAME,
                          inactivity_timeout=self._INACTIVITY_TIMEOUT,
                          resource_base_path='/com/hack_computer/Clubhouse')
+        Portal.request_background()
```

Comment:
Okay, I decided I wanted more specifics because it's bothering me for multiple reasons :b Poking at this with Bustle I see…

![Screenshot from 2020-11-25 13-27-41](https://user-images.githubusercontent.com/132063/100283263-06897f80-2f22-11eb-870a-7d5bc582495c.png)

So, the background request fails (asynchronously) because `org.freedesktop.impl.portal.Access.AccessDialog` fails, saying "Only the focused app is allowed to show a system access dialog".